python-sdk/rebuff/rebuff_pythonsdk.py

```diff
@@ -0,0 +1,228 @@
+import secrets
+from typing import Any, Optional, Tuple, Union
+from detect_pi_heuristics import detect_prompt_injection_using_heuristic_on_input
+from detect_pi_vectorbase import init_pinecone, detect_pi_using_vector_database
+from detect_pi_openai import render_prompt_for_pi_detection, call_openai_to_detect_pi
+from pydantic import BaseModel
+import langchain
+
+
+class Rebuff_Detection_Response(BaseModel):
```

Comment:
Naming convention is to not use underscores in class names: 
```suggestion
class RebuffDetectionResponse(BaseModel):
```